clients/client-s3/test/e2e/S3.browser.e2e.spec.ts

```diff
@@ -69,10 +93,7 @@ describe("@aws-sdk/client-s3", () => {
 
     const buf = createBuffer("1KB");
 
-    // TODO(vitest)
-    // Caused by: RequestContentLengthMismatchError: Request body length does not match content-length header
-    // only in vitest + happy-dom.
-    it.skip("should succeed with blob body", async () => {
```

Comment:
upgrading vitest and happy-dom, blob body started working and ReadableStream stopped working 🙃